File: Leetcode/Leet0189.java
class Solution {
    public int[] rotate(int[] nums, int k) {
        int arr[] = new int[nums.length];
        int j = 0;
        k = k%nums.length;
        for(int i = k;i < nums.length; i++){
            arr[i] = nums[j];
            j += 1;
        }
        for(int i = 0; i < k; i++){
            arr[i] = nums[j];
            j += 1;
        }
        for(int i = 0; i < nums.length; i++){
            nums[i] = arr[i];
        }
        return nums;
    }
}

File: Leetcode/Leet0189.py
https://leetcode.com/problems/rotate-array/

class Solution:
    def rotate(self, nums: List[int], k: int) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        for i in range(k%len(nums)):
            nums.insert(0,nums[-1])
            del nums[-1]
        return nums